### PR TITLE
Homework CSS part 2

### PR DESCRIPTION
**Pull request by TaniaKho**
Please review my homework.
Thank you!

### CODE REVIEW

**Comment by pluhin** [comment]
CSS is file, but check your forms,  there are a lot of issue with labels design

**Review by pluhin — CHANGES_REQUESTED** [review]
Could you check all labels

**Review by pluhin — APPROVED** [review]

File: tkhomich/04_css/CSS_Python_2/Python 2.html [hunk 1]
```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<HTML>
+<HEAD>
+    <META charset="utf-8">
+    <TITLE>Программа на Python 2</TITLE>
+</HEAD>
+<BODY>
+     <!-- Вариант 2 -->
+     <style media="screen">
```
Comment (pluhin) [inline tkhomich/04_css/CSS_Python_2/Python 2.html]: What tag style does here? It should be in tag head

File: tkhomich/04_css/CSS_Python_2/Python 2.html [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+            body {
+                font-family: sans-serif;
+                font-size: 14px;
+                background-color: #eee;
+            }
+        </style>
+        <style media="print">
```
Comment (pluhin) [inline tkhomich/04_css/CSS_Python_2/Python 2.html]: Sames as above

File: tkhomich/04_css/CSS_Python_2/Python 3.html [hunk 3]
```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<HTML>
+<HEAD>
+    <META charset="utf-8">
+    <TITLE>Программа на Python 3</TITLE>
+</HEAD>
+<BODY>
+     <!-- Вариант 2 -->
+     <style media="screen">
```
Comment (pluhin) [inline tkhomich/04_css/CSS_Python_2/Python 3.html]: Tag style should be in head

File: tkhomich/04_css/CSS_Python_3/Python 1.html [hunk 4]
```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<HTML>
+<HEAD>
+    <META charset="utf-8">
+    <TITLE>Программа на Python</TITLE>
+</HEAD>
+<BODY>
+    <!-- Вариант 3 -->
+    <LINK REL="stylesheet" HREF="stylesS.css" TYPE="text/css" media="screen">
```
Comment (pluhin) [inline tkhomich/04_css/CSS_Python_3/Python 1.html:7]: Link tag should be in tag head

File: tkhomich/04_css/CSS_Python_3/Python 2.html [hunk 5]
```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<HTML>
+<HEAD>
+    <META charset="utf-8">
+    <TITLE>Программа на Python 2</TITLE>
+</HEAD>
+<BODY>
+    <!-- Вариант 3 -->
+    <LINK REL="stylesheet" HREF="stylesS.css" TYPE="text/css" media="screen">
```
Comment (pluhin) [inline tkhomich/04_css/CSS_Python_3/Python 2.html:7]: Move it into tag head

File: tkhomich/04_css/CSS_Python_3/Python 3.html [hunk 6]
```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<HTML>
+<HEAD>
+    <META charset="utf-8">
+    <TITLE>Программа на Python 3</TITLE>
+</HEAD>
+<BODY>
+    <!-- Вариант 3 -->
+    <LINK REL="stylesheet" HREF="stylesS.css" TYPE="text/css" media="screen">
```
Comment (pluhin) [inline tkhomich/04_css/CSS_Python_3/Python 3.html:7]: To head

**Review by pluhin — CHANGES_REQUESTED** [review]

File: tkhomich/04_css/CSS_Python_2/Python 2.html (hunk 1 above)
Comment (TaniaKho) [reply tkhomich/04_css/CSS_Python_2/Python 2.html]: Updated. Moved to Head.

File: tkhomich/04_css/CSS_Python_2/Python 2.html (hunk 2 above)
Comment (TaniaKho) [reply tkhomich/04_css/CSS_Python_2/Python 2.html]: Updated. Moved to Head.

File: tkhomich/04_css/CSS_Python_2/Python 3.html (hunk 3 above)
Comment (TaniaKho) [reply tkhomich/04_css/CSS_Python_2/Python 3.html]: Updated. Moved to Head.

File: tkhomich/04_css/CSS_Python_3/Python 1.html (hunk 4 above)
Comment (TaniaKho) [reply tkhomich/04_css/CSS_Python_3/Python 1.html:7]: Updated. Moved to Head.

File: tkhomich/04_css/CSS_Python_3/Python 2.html (hunk 5 above)
Comment (TaniaKho) [reply tkhomich/04_css/CSS_Python_3/Python 2.html:7]: Updated. Moved to Head.

File: tkhomich/04_css/CSS_Python_3/Python 3.html (hunk 6 above)
Comment (TaniaKho) [reply tkhomich/04_css/CSS_Python_3/Python 3.html:7]: Updated. Moved to Head.

**Pull request merged**